mozetl/engagement/churn/__init__.py

```diff
@@ -1 +1 @@
-from main import *  # NOQA
+from job import *  # NOQA
```

Comment:
Ugh, this is a red flag, start imports are considered an anti-pattern since they hide the underlying implementation and confuse debuggers. If it's just to save typing "job" in import statements, I think it's fair to stop providing this shim.

Reply:
:+1:

The initial idea was to be able to import the main entry point from churn and retention in the engagement module without having to snake through the entire project to get to it.

Originally, this used to be called as `from mozetl.churn import churn; churn.main`. It then got refactored into an 'engagement/churn/main.py` file. I thought the repetition of the module and script name was redundant and tried to import the entry point into module scope. I hadn't figured out what to call the file with the `extract`, `transform`, and `save`/`load` functions yet, so I left it as `main.py`.

I did happen to reuse `extract` in daily retention, so I imported it into module scope along with everything else in `main.py`. In retrospect, muddying up the namespace doesn't seem like a worthwhile trade-off.

It would be nice to export a stable api here (but I don't have a concrete idea on what that would look like).